cmdutil/testdata/stubcmd.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+)
+
+var sigmap = map[string]os.Signal{
+	"INT":  os.Interrupt,
+	"TERM": syscall.SIGTERM,
+}
+
+func main() {
+	var (
+		trap     = flag.String("trap", "", "signals")
+		exit     = flag.Int("exit", 0, "exit status")
+		trapExit = flag.Int("trap-exit", 0, "exit status when trapping signal")
+		sleep    = flag.Float64("sleep", 0, "sleep seconds")
```

Comment:
why not int64?

Reply:
Because, I wanted to specify sleep seconds as a decimal number like `-sleep 0.3`.

Reply:
I prefer to use `flag.Duration` in this case. This is easy to specify seconds (-sleep 3s) or hours/minutes (-sleep 2h45m). But this is problem that is likes and dislikes. And this is just nit-picker. :)

Reply:
Oh, I did not know that `flag.Duration` could take `ms`. I Changed to use flag.Duration.